app/models/assignment_repo/creator.rb

```diff
@@ -189,5 +190,63 @@ def suffixed_repo_name(repository_name, suffix_count)
       suffix = "-#{suffix_count}"
       repository_name.truncate(100 - suffix.length, omission: '') + suffix
     end
+
+    # Handles assignment repository setup
+    #
+    # configs_repo    - GitHubRepository containing the configuration files
+    # assignment_repo - GitHubRepository for which to perform the configuration
+    #                   setups
+    #
+    # Returns nothing
+    def setup_repository(configs_repo, assignment_repo)
+      return unless configs_repo.classroom_configs?
+      conf_tree = configs_repo.branch('github-classroom').commit.commit.tree.sha
+      configs_repo.tree(conf_tree).tree.each do |conf|
```

Comment:
Could we assign this to a variable with a descriptive name? Iterating over the tree of the conf_tree of the conf_repo is pretty confusing